lib/players/human.rb

```diff
@@ -0,0 +1,12 @@
+module Players
+  class Human < Player
+
+#The human player must implement a #move method that takes in a board argument and allows a human player to enter a move via the CLI.
+#The method should return the value the user enters.
+# Even though the method accepts a board argument, it does not need to use it.
+
+    def move(board)
+      input = gets.strip
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(board)
  puts "Where would you like to move?"
  gets.chomp
end
```